src/js/components/form-elements/v2/style.scss

```diff
@@ -214,6 +214,10 @@ input, .date-field-input {
       }
     }
   }
+
+  .counted-by-width {
```

Comment:
This fits more in the [cycleCount.scss](https://github.com/openboxes/openboxes/blob/develop/src/js/components/cycleCount/cycleCount.scss). And change the naming the be a little bit more generic, in case it needs to be expanded with other properties